fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -80,13 +80,38 @@ def self.run(params)
         version_number = line.partition('=').last
         return version_number if Helper.is_test?
 
+        targets_available = self.parse_targets(results)
+        self.validate_target(params, targets_available)
```

Comment:
Can `self.validate_target` be renamed to `self.validate_targets` and it we could just pass it the `results` and it could handle calling `parse_targets`? Just to condense this down into one line that is validating targets.